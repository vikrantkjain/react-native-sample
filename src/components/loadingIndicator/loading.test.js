import renderer from 'react-test-renderer';
import LoadingIndicator from './index';

test('Test loadingIndicator', () => {
  const tree = renderer.create(<LoadingIndicator isLoading={true} />).toJSON();
  expect(tree).toMatchSnapshot();
});
