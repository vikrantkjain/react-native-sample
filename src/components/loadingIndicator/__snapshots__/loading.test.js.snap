// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test loadingIndicator 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "elevation": 20,
      "flex": 1,
      "height": "100%",
      "justifyContent": "center",
      "position": "absolute",
      "width": "100%",
    }
  }
>
  <View
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 50,
          "height": 100,
          "justifyContent": "center",
          "width": 100,
        },
        undefined,
      ]
    }
  >
    <ActivityIndicator
      color="#048de0"
      size={80}
    />
  </View>
</View>
`;
